backend/benefit/applications/models.py

```diff
@@ -174,6 +174,7 @@ def with_due_instalments(self, status: InstalmentStatus):
         """Query applications with instalments with past due date and a specific status."""
         return (
             self.filter(
+                status=ApplicationStatus.ACCEPTED,
```

Comment:
added this check so that only accepted applications with instalments are sent to talpa